pkgs/top-level/lua-packages.nix

```diff
@@ -145,11 +146,12 @@ let
   };
 
   luasec = buildLuaPackage rec {
-    version = "0.5";
-    name = "sec-${version}";
-    src = fetchzip {
-      url = "https://github.com/brunoos/luasec/archive/luasec-${version}.tar.gz";
-      sha256 = "1zl6wwcyd4dfcw01qan7dkcw0rgzm69w819qbaddcr2ik147ccmq";
+    name = "sec-git-2015-04-17";
```

Comment:
Thanks! Fixed and rebased.

Now that I'm thinking about it, would it maybe make sense to use `0.5-git-2015-04-17` as version for clarity and upgrade-ability?